plotly/utils.py

```diff
@@ -13,6 +15,9 @@
 
 import pytz
 
+
+from . exceptions import PlotlyError
```

Comment:
this was some stuff I had to do to get around circular imports across py 27 and py 33